bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2800,6 +2781,7 @@ ReplicaImp::ReplicaImp(const ReplicaConfig &config,
     ps_->endWriteTran();
   }
 
+  LOG_INFO(GL, "Starting internal replica thread pool with 8 threads.");
```

Comment:
Extract a local variable, in order to future-proof the log statement

Reply:
Sounds good.